js/bybit.js

```diff
@@ -1610,16 +1672,213 @@ module.exports = class bybit extends Exchange {
         //          }
         //     }
         //
+        // unified margin
+        //
+        //     {
+        //         "retCode": 0,
+        //         "retMsg": "OK",
+        //         "result": {
+        //             "category": "linear",
+        //             "list": [
+        //                 {
+        //                     "symbol": "BTCUSDT",
+        //                     "bidPrice": "19255",
+        //                     "askPrice": "19255.5",
+        //                     "lastPrice": "19255.50",
+        //                     "lastTickDirection": "ZeroPlusTick",
+        //                     "prevPrice24h": "18634.50",
+        //                     "price24hPcnt": "0.033325",
+        //                     "highPrice24h": "19675.00",
+        //                     "lowPrice24h": "18610.00",
+        //                     "prevPrice1h": "19278.00",
+        //                     "markPrice": "19255.00",
+        //                     "indexPrice": "19260.68",
+        //                     "openInterest": "48069.549",
+        //                     "turnover24h": "4686694853.047006",
+        //                     "volume24h": "243730.252",
+        //                     "fundingRate": "0.0001",
+        //                     "nextFundingTime": "1663689600000",
+        //                     "predictedDeliveryPrice": "",
+        //                     "basisRate": "",
+        //                     "deliveryFeeRate": "",
+        //                     "deliveryTime": "0"
+        //                 }
+        //             ]
+        //         },
+        //         "retExtInfo": null,
+        //         "time": 1663670053454
+        //
         const result = this.safeValue (response, 'result', []);
         let rawTicker = undefined;
         if (Array.isArray (result)) {
             rawTicker = this.safeValue (result, 0);
         } else {
-            rawTicker = result;
+            if (isV3) {
```

Comment:
@sc0Vu spot v3 does not have `list` so this is breaking rn 
![image](https://user-images.githubusercontent.com/43336371/201886611-2cfc44c4-5518-4a7f-b1f2-03969b33d31d.png)